### PR TITLE
Enable support for multiple seeAlsos in manifest.

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-ruby-2.3.1
+ruby-2.4.3
```
Comment (hackmastera) [inline .ruby-version]: Is this the version on the server? It might be nice to have this in a separate commit.

Comment (tpendragon) [reply .ruby-version]: Sure, split out to a new commit.

Comment (hackmastera) [reply .ruby-version]: Thank you! And thank you for fixing it.
